### ISSUE TITLE
Background event background color does not fill cells in Chrome

### ISSUE THREAD

**Issue by robynluo** [fullcalendar/fullcalendar#4145]
In the month view, I created a lot of background-resource-event.
There will be a problem, see below：
![image](https://user-images.githubusercontent.com/8199379/38669135-29b2f0c4-3e78-11e8-9e17-d2850ec6ec15.png)

I simulated the scene.
jsbin url: 
https://jsbin.com/xesowok/1/edit?html,css,js,console,output

![image](https://user-images.githubusercontent.com/8199379/38672927-3159311c-3e82-11e8-8dec-99ff9b3be57c.png)

The td width of each table can't be consistent. When adjusting the width of the browser, sometimes the correct situation occurs.

![image](https://user-images.githubusercontent.com/8199379/38673763-cbd8d4d4-3e84-11e8-8f5d-9434a436cd16.png)

I hope this problem can be solved, thank you.

**Comment by acerix** [comment]
Thanks for the report.  I can see the issue in Chrome, but in Firefox it always looks good.  

Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by arshaw** [comment]
fullcalendar.js has been patched to fix this:

[fullcalendar-0.0.0.zip](https://github.com/fullcalendar/fullcalendar/files/1947922/fullcalendar-0.0.0.zip)

**Comment by robynluo** [comment]
@arshaw 
Thanks for you support.

I've replaced fullcalendar.js in my demo. it looks good.

![image](https://user-images.githubusercontent.com/8199379/39391038-234dd744-4ad0-11e8-990a-db86afc9ecee.png)

![image](https://user-images.githubusercontent.com/8199379/39391043-34f52592-4ad0-11e8-92c3-2c2aea819068.png)

**Comment by arshaw** [comment]
great. ill leave this ticket open until i launch it in a real release.

**Comment by arshaw** [comment]
fixed with 312b5433d1ef2215453945498ef48f4b0c7c2356

will comment again when released

**Comment by arshaw** [comment]
released in [3.10.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.10.0)

**Comment by kamleshwebtech** [comment]
I am using full calendar version 4. You can make it as event and pass rendering and color params to calelendar:
```
$eventRecords = array();
$eventRecords[0]['title'] = 'Available';
$eventRecords[0]['start'] = '2019-11-09T21:00:00';
$eventRecords[0]['end'] = '2019-11-09T22:00:00';
$eventRecords[0]['overlap'] = false;
$eventRecords[0]['rendering'] = 'background';
$eventRecords[0]['color'] = '#00ff00';
```
It works for me. I hope, this will help you also. Thanks for asking this question.
